### ISSUE TITLE
Discourse - Missing Error message of the Gebruikersnaam input field in the Registreren modal

### ISSUE THREAD

**Issue by savitris** [frameless/developer.overheid.nl#140]
**Dit probleem kan op meerdere pagina's aanwezig zijn, waaronder:**
<a href="https://community.developer.overheid.nl/t/officiele-bekendmakingen-binnenlandse-zaken-en-koninkrijksrelaties/726">Officiële bekendmakingen (Binnenlandse Zaken en Koninkrijksrelaties)</a>

**Probleembeschrijving:**
Probleem: Bij het klikken op de "Aanmelden" knop onderaan het Registreren formulier (wanneer je de verplichte velden leeg laat) verschijnt er een foutmelding onder 2 van de 3 verplichte invoervelden. Het "Gebruikersnaam" veld krijgt echter geen foutmelding. Foutmeldingen zouden consistent voor alle verplichte invoervelden moeten worden weergegeven.
Alleen als je al iets in het "Gebruikersnaam" veld hebt getypt en het daarna weer verwijdert, wordt de foutmelding "Voer een gebruikersnaam in" zichtbaar.

Mogelijke verbeteringen: Voeg `aria-required="true"` toe aan alle verplichte invoervelden. Voeg ook de tekst "Verplicht" toe naast het sterretje (of leg bovenaan het formulier uit wat het sterretje betekent). Zorg er ten slotte voor dat de foutmelding voor het "Gebruikersnaam" veld op dezelfde manier verschijnt als bij de andere verplichte velden.

**Referentiecode:**
```
<div id="username-validation" class="tip bad ember-view">  <svg class="fa d-icon d-icon-times svg-icon svg-string" xmlns="http://www.w3.org/2000/svg"><use href="#times"></use></svg>
  Voer een gebruikersnaam in
</div>
```

**Gerelateerde WCAG:**
WCAG 3.3.3 Error Suggestion
Level AA

**Referentieafbeelding:**

![A form has some input fields that show an error message complaining about them not being filled in](https://github.com/user-attachments/assets/b0140c24-145e-4e0e-b4ec-6720bfa6b446)

_**Referentienummer:** A151_
